README.md

```diff
@@ -1,444 +1,86 @@
-final_sign_installation|final sign installation date|text
-locations_fb|location of flashing beacon|text
-locations_wyss|location of watch your speed sign|text
-
-## `wys`: Watch Your Speed Signs
-
-The city has installed [Watch Your Speed Signs](https://www.toronto.ca/services-payments/streets-parking-transportation/road-safety/vision-zero/safety-initiatives/initiatives/watch-your-speed-program/) that display the speed a vehicle is travelling at and flashes if the vehicle is travelling over the speed limit. Installation of the sign was done as part of 3 programs: the normal watch your speed sign program, mobile watch your speed which has signs mounted on trailers that move to a different location every few weeks, and school watch your speed which has signs installed at high priority schools. As part of the [Vision Zero Road Safety Plan](https://www.toronto.ca/services-payments/streets-parking-transportation/road-safety/vision-zero/), these signs aim to reduce speeding. 
-
-The [`wys/api`](wys/api) folder contains a Python script that pulls the data from a cloud API daily as well as the sql structure to aggregate the data.
-
-### Data Elements
-
-The data is inserted into `wys.raw_data`. Data from the API is already pre-aggregated into roughly 5 minute bins.
-
-|Field name|Data type|Description|Example|
-|------|------|-------|------|
-`raw_data_uid`|integer|A unique identifier for the `raw_data` table|2655075
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`datetime_bin`|timestamp|Start time of the bin|2018-10-29 10:00:00
-`speed`|integer|Exact speed of the number of vehicles in `count`|47
-`count`|integer|Number of vehicles in datetime_bin/api_id/speed combination|2
-`counts_15min`|integer|A unique identifier for `counts_15min` table. Indicates if the data has already been processed or not.|150102
+[`volumes/rescu/`](volumes/rescu/)
 
-`wys.counts_15min` has aggregated 15 minute time bins, and aggregated 5 km/h speed bins by using the `aggregate_speed_counts_15min()` function. Values for the speed bins are replaced by lookup table IDS.
+AKA Vehicle Detector Station (VDS)
 
-|Field name|Data type|Description|Example|
-|------|------|-------|------|
-`counts_15min`|integer|A unique identifier for the `counts_15min` table|2655075
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`datetime_bin`|timestamp|Start time of the 15 minute aggregated bin|2018-10-29 10:00:00
-`speed_id`|integer|A unique identifier for the 5 minute speed bin in the `speed_bins` table|5
-`count`|integer|Number of vehicles in datetime_bin/api_id/speed bin combination|7
+Road Emergency Services Communication Unit (RESCU) data tracks traffic volume on expressways using loop detectors. 
+More information can be found on the [city's website](https://www.toronto.ca/services-payments/streets-parking-transportation/road-restrictions-closures/rescu-traffic-cameras/) 
```

Comment:
Does this link make people think the data come from cameras?

Reply:
I didn't write this; I just moved it from elsewhere in the same readme.